client/src/models/Report.js

```diff
@@ -566,16 +585,24 @@ export default class Report extends Model {
     )
   }
 
+  static isEngagementAllDay(report) {
+    return !report.duration
```

Comment:
Seeing as it's possible to enter a duration of `0` (zero); shouldn't this only return true when duration is `null` or `undefined`? Alternatively, make the duration input only accept positive integers (e.g. arrow down should either not decrement below 1, or perhaps when decrementing from 1 empty out the field).
Note that conflict detection only checks for `duration === null`!

Reply:
Hmm yes, we should prevent `duration = 0`